contents/teams/error-tracking/objectives.mdx

```diff
@@ -1,27 +1,56 @@
-### Q3 2025 Objectives
-
-#### SDK improvements and language support
-* Mobile support
-   * React Native JS (& Hermes source maps)
-   * Native iOS & Android
-   * Stretch: Flutter
-* Add error tracking to 3 more server side SDKs
-* Complete `posthog-js` and `posthog-js-lite` merge to a monorepo
-  * Stretch: continue with [the megaissue](https://github.com/PostHog/posthog-js/issues/2067) as time permits
-
-#### Integrate with other products
-* Find issues that impact signup / activation / retention heavily (correlation query)
-* Use Data warehouse / Revenue analytics to order issues by revenue impact
-* Add session context to issue page (example: breadcrumbs)
-* AI: Add session summaries to issues
-
-#### Product experience
-* Gather more qualitative data
-  * Continue customer interviews
-  * Make sure existing customers remain happy
-* Make in-app searching more intuitive
-  * Add error tracking tab to taxonomic filter
-  * Add email to omnisearch
-  * Expose search via Max AI
-* Clearly call out issues with source maps and offer resolution steps
-* Support Docs team's goal to make error tracking docs amazing
\ No newline at end of file
+### Q4 2025 Objectives
+
+#### Go all in on AI (Driver: <TeamMember name="David Newell" />)
+* Implement issue autofixing
+  * Integrate with Tasks product for in-app experience
+  * Build terminal based experience within the CLI
+    * Add issue selection to MCP (e.g. by file, most impactful)
+    * Test generation to prevent regressions
+* Add exception capture auto instrumentation to the wizard
+* Integrations to capture exceptions for popular LLM / agent tools (e.g. Langchain)
+
+#### Ship mobile error tracking (Drivers: <TeamMember name="Oliver Browne" />, <TeamMember name="Manoel Aranda Neto" />)
+* [Sequencing TBD]
```

Comment:
Previous discussion in https://github.com/PostHog/posthog.com/pull/12006#discussion_r2173159400

Reply:
we can define this once @ioannisj is back, then Olly Ioannis and I will sit and update this so we dont block this PR